From d178058028050aeec649c85644cc9eed08f0aa37 Mon Sep 17 00:00:00 2001
From: sezero <sezero@users.sourceforge.net>
Date: Sun, 15 Jan 2017 01:56:56 +0300
Subject: [PATCH] libFLAC/cpu.c: Replace memory.h include with string.h

cpu.c was the only source to use memory.h instead of string.h.

Signed-off-by: Erik de Castro Lopo <erikd@mega-nerd.com>
---
 src/libFLAC/cpu.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libFLAC/cpu.c b/src/libFLAC/cpu.c
index 1c80a728..6629fb95 100644
--- a/src/libFLAC/cpu.c
+++ b/src/libFLAC/cpu.c
@@ -37,7 +37,7 @@
 #include "private/cpu.h"
 #include "share/compat.h"
 #include <stdlib.h>
-#include <memory.h>
+#include <string.h>
 
 #if defined(_MSC_VER)
 #  include <intrin.h> /* for __cpuid() and _xgetbv() */
-- 
2.11.0

